src/Charcoal/Email/Email.php

```diff
@@ -707,17 +708,21 @@ public function send(): bool
             $mail->Subject = $this->subject();
             $mail->Body    = $this->msgHtml();
             $mail->AltBody = $this->msgTxt();
-
-            $ret = $mail->send();
-
-            if ($this->logEnabled() === true) {
-                $this->logSend($ret, $logId, $mail);
-            }
         } catch (Exception $e) {
-            $ret = false;
             $this->logger->error(
                 sprintf('Error sending email: %s', $e->getMessage())
             );
+            return false;
+        }
+
+        try {
+            $ret = $mail->send();
+        } catch (\PHPMailer\PHPMailer\Exception $e) {
```

Comment:
Import the exception as:

```diff
  // From 'phpmailer/phpmailer'
+ use PHPMailer\PHPMailer\Exception as PHPMailerException;
  use PHPMailer\PHPMailer\PHPMailer;
```

```suggestion
        } catch (PHPMailerException $e) {
```